src/app/components/editors/gameobject/gameobject-template/gameobject-template.component.html

```diff
@@ -0,0 +1,104 @@
+<app-top-bar [handler]="handlerService"></app-top-bar>
+
+<div class="container-fluid">
+
+  <span *ngIf="editorService.loading">Loading...</span>
+
+  <div *ngIf="editorService.form && !!editorService.loadedEntityId && !editorService.loading">
+
+    <app-query-output [editorService]="editorService" (executeQuery)="editorService.save($event)"></app-query-output>
+
+    <hr class="mt-2 mb-1">
+
+    <div class="wiki-link mb-2">
+      <a [href]="docUrl" target="_blank">
+        <i class="fas fa-link"></i> {{ editorService.entityTable }} documentation
+      </a>
+    </div>
+
+    <form
+      [formGroup]="editorService.form"
+      class="form-group edit-form"
+    >
+
+      <div class="row">
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="entry">entry</label>
+          <i
+            class="fas fa-info-circle ml-1" [placement]="'auto'"
+            [tooltip]="'The unique ID of the gameobject'"
+          ></i>
+          <input [formControlName]="'entry'" id="entry" type="number" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="name">name</label>
+          <input [formControlName]="'name'" id="name" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="type">type</label>
+          <app-single-value-selector-btn
+            [control]="editorService.form.get('type')"
+            [config]="{ options: GAMEOBJECT_TYPE, name: 'type' }"
+            [modalClass]="'modal-md'"
+          ></app-single-value-selector-btn>
+          <input [formControlName]="'type'" id="type" type="number" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="displayId">displayId</label>
+          <i
+            class="fas fa-info-circle ml-1" [placement]="'auto'"
+            [tooltip]="'Graphic model id sent to the client from GameObjectDisplayInfo.dbc'"
+          ></i>
+          <input [formControlName]="'displayId'" id="displayId" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="IconName">IconName</label>
+          <app-single-value-selector-btn
+            [control]="editorService.form.get('IconName')"
+            [config]="{ options: GAMEOBJECT_ICON, name: 'IconName' }"
+            [modalClass]="'modal-xl'"
```

Comment:
the data in the modal is quite small, no need to pass such a big modal like `modal-xl`. Better a smaller one like `modal-md` for example